packages/netlify-cms-backend-github/src/API.ts

```diff
@@ -547,65 +568,22 @@ export default class API {
     }
   }
 
-  matchingEntriesFromDiffs(diffs: Octokit.ReposCompareCommitsResponseFilesItem[]) {
-    // media files don't have a patch attribute, except svg files
-    const matchingEntries = diffs
-      .filter(d => d.patch && !d.filename.endsWith('.svg'))
-      .map(f => ({ path: f.filename, newFile: f.status === 'added' }));
-
-    return matchingEntries;
-  }
-
-  async retrieveMetadata(contentKey: string) {
```

Comment:
This is now at `backend.js` which also makes it easier to handle empty diffs (reads the original entry in that case)